recipes/ffnvcodec-headers/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = 'ffnvcodec-headers' %}
+{% set version = '8.2.15.4' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://github.com/FFmpeg/nv-codec-headers/releases/download/n{{ version }}/nv-codec-headers-{{ version }}.tar.gz'
+  sha256: 9a567b45ece1930e65803411f12736d8925f0d1adbc31e962cb5c441969940f0
+
+build:
+  number: 1
```

Comment:
We usually start from `0`. (Not a big deal though.)

Reply:
Dam. Sorry :/